documentation/sdkautomation/GenerateOutputSchema.json

```diff
@@ -46,6 +46,19 @@
             "type": "string"
           }
         },
+        "typespecProject": {
```

Comment:
I found js sdk automation also output `packageFolder`, is it needed somwwhere?

Reply:
This is covered by `path` property.

Reply:
Just to confirm, so we don't need to output `packageFolder` anymore, right?